htdocs/core/class/extrafields.class.php

```diff
@@ -1065,6 +1065,7 @@ public function showInputField($key, $value, $moreparam = '', $keysuffix = '', $
 
 			$out .= '<select class="flat '.$morecss.' maxwidthonsmartphone" name="'.$keyprefix.$key.$keysuffix.'" id="'.$keyprefix.$key.$keysuffix.'" '.($moreparam ? $moreparam : '').'>';
 			$out .= '<option value="0">&nbsp;</option>';
+			$out .= '<option value="-2">- '. $langs->trans("SelectEmpty") . ' -</option>';
```

Comment:
I think the "select empty" option should not appear when you are on a card (modifying the value of the extrafield), but only for list filters.

Reply:
Well spotted ! I fixed it